README.md

```diff
@@ -77,6 +77,13 @@ To run LIVE bundles on localhost:
 - Then run `rm -rf build && npm run build:live && npm run start`
 - Visit a live article: http://localhost:7080/news/articles/c8xxl4l3dzeo
 
+## Changing request location
+
+Some features perform differently dependant on whether a user is located within the UK or internationally. You can explicitly request a specific version by accessing Simorgh via a specific localhost BBC domain:
+
```

Comment:
For this to work, I believe we'd need to add a hosts file entry (`/etc/hosts` or `C:\Windows\System32\drivers\etc\hosts`)

```
127.0.0.1 localhost.bbc.co.uk
```